docs/framework/wcf/how-to-implement-a-wcf-contract.md

```diff
@@ -78,7 +78,7 @@ This is the second of six tasks required to create a basic [!INCLUDE[indigo1](..
     }  
     ```  
   
-    ```  
+    ```vbnet
```

Comment:
our build system supports vb instead of vbnet for the fenced code blocks. For more info, see https://github.com/dotnet/docs/blob/master/styleguide/template.md#supported-languages